Snabble/Core/API/SnabbleAPI.swift

```diff
@@ -102,35 +115,48 @@ public enum SnabbleAPI {
 
     private static var providerPool = [Identifier<Project>: ProductProvider]()
 
-    public static var certificates: [GatewayCertificate] {
+    /// Gateway certificates for payment routes
+    static var certificates: [GatewayCertificate] {
         return self.metadata.gatewayCertificates
     }
 
+    /// Available projects after an successful setup
     public static var projects: [Project] {
         return self.metadata.projects
     }
 
+    /// Additional information provided by Snabble
     public static var flags: Flags {
         return self.metadata.flags
     }
 
+    /// API links for snabble features
     public static var links: MetadataLinks {
         return self.metadata.links
     }
 
+    /// Terms of Use for the Snabble App
     public static var terms: Terms? {
         return self.metadata.terms
     }
 
+    /// Are used to combine multiple projects
     public static var brands: [Brand] {
         return self.metadata.brands ?? []
     }
 
+    /// Finds first project for a given id
+    /// - Parameter projectId: matching id
+    /// - Returns: `Project` or `nil` if none was found
     public static func project(for projectId: Identifier<Project>) -> Project? {
         return self.metadata.projects.first { $0.id == projectId }
     }
 
-    public static func setup(_ config: SnabbleAPIConfig, completion: @escaping () -> Void ) {
+    /// First method to be called to verify the initialization of the `SnabbleSDK`
```

Comment:
verify könnte irreführend sein, warum nicht einfach ".. to be called to initialize..."

Reply:
e9768f59b693a08542b53878a27bebd4b676c334